packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -77,7 +88,15 @@ export abstract class DeployRetrieveExecutor<
         status === RequestStatus.SucceededPartial
       );
     } catch (e) {
-      throw formatException(e);
+      if (e.name === 'SourceConflictError') {
+        this.handleSourceConflictError(e);
+        return true;
```

Comment:
Why do we return `true` here instead of one of the `RequestStatus` enums?

Reply:
Well, at this point the request did not succeed.  It threw an exception which was caught and handled.  So I think the true response is like, yep it ran.

If a status were returned, I'm not sure which one it would be:
<img width="395" alt="2023-03-21_15-50-40" src="https://user-images.githubusercontent.com/46458081/226759403-ca7b91d8-dbd7-47a0-a7df-dc2ceac731de.png">

Reply:
This one might be good to discuss at 11sies if it could potentially have a meaningful impact in terms of code or functionality. For instance, when Apex tests run but return failures we show something like "the command failed to run", and in this case, it did fail to complete the deploy/retrieve operation because of those conflicts, right?

Reply:
Well the difference here is that the user has the whole conflict experience in the UI where they choose how to "finish" the operation.  So an extra notification that the deploy op didn't succeed might not be needed.
<img width="405" alt="2023-03-22_10-58-10" src="https://user-images.githubusercontent.com/46458081/226995761-aa5f2f80-24ee-47ac-ab5d-2ba6754a21ff.png">